cpp/pybind/visualization/rendering/rendering.cpp

```diff
@@ -516,6 +516,24 @@ void pybind_rendering_classes(py::module &m) {
                  "The flags should be ORed from Scene.UPDATE_POINTS_FLAG, "
                  "Scene.UPDATE_NORMALS_FLAG, Scene.UPDATE_COLORS_FLAG, and "
                  "Scene.UPDATE_UV0_FLAG")
+            .def("remove_geometry", &Scene::RemoveGeometry,
```

Comment:
Add annotations for arguments for all python bindings. This is necessary for proper documentation from Sphinx, as well as for default values.

e.g.:
```cpp
.def("add_geometry",
                 (bool (Scene::*)(
                         const std::string &, const t::geometry::Geometry &,
                         const MaterialRecord &, const std::string &, size_t)) &
                         Scene::AddGeometry,
                 "name"_a, "geometry"_a, "material"_a,
                 "downsampled_name"_a = "", "downsample_threshold"_a = SIZE_MAX,
                 "Adds a Geometry with a material to the scene")
```

Reply:
Done. Added to all `Scene` pybinds.